example/generator/odp_generator.c

```diff
@@ -765,11 +765,12 @@ static int gen_recv_thread(void *arg)
  */
 static void print_global_stats(int num_workers)
 {
-	odp_time_t cur, wait, next;
+	odp_time_t cur, wait, next, left;
 	uint64_t pkts_snd = 0, pkts_snd_prev = 0;
 	uint64_t pps_snd = 0, maximum_pps_snd = 0;
 	uint64_t pkts_rcv = 0, pkts_rcv_prev = 0;
 	uint64_t pps_rcv = 0, maximum_pps_rcv = 0;
+	uint64_t stall = 0;
```

Comment:
Nit: Does GCC flag this as an uninitialized variable without the initialization here? That doesn't seem needed.

Reply:
No there's no warning if we leave that uninitialized